test/integration/test-extensions-loading.js

```diff
@@ -21,8 +21,6 @@ function checkElementUpgrade(element) {
   expect(element).to.have.class('-amp-element');
   expect(element).to.have.class('-amp-layout-responsive');
   expect(element).to.have.class('-amp-layout-size-defined');
-  expect(element).to.not.have.class('-amp-notbuilt');
```

Comment:
Ok. brought back the build during discover - and changed the tests to wait on multiple events for the elements numbers. This has passed but the test flaked - rerunning the test for this but it should be good to go.